utils.py

```diff
@@ -0,0 +1,42 @@
+def login_required(func):
```

Comment:
This is a common practice. Here I created a general utilities file, since there's only 3 functions that need to be moved out of the views. Of course you can take this as far as you need to -- if you had a bunch of functions that only dealt with sorting and building Pokemen, then you'd make a `pokemon.py` module to handle those. Same goes for any specialized code that doesn't really belong in your views.